com/steuerbetrag.go

```diff
@@ -12,9 +13,11 @@ import (
 type Steuerbetrag struct {
 	Steuerkennzeichen steuerkennzeichen.Steuerkennzeichen `json:"steuerkennzeichen,omitempty" validate:"required" example:"UST19"` // Kennzeichnung des Steuersatzes, bzw. Verfahrens
 	// Basiswert and Steuerwert are _not_ marked as required because the steuerwert 0 is actually valid
-	Basiswert  decimal.Decimal             `json:"basiswert" example:"100"`                              // Basiswert ist der Nettobetrag für den die Steuer berechnet wurde
-	Steuerwert decimal.Decimal             `json:"steuerwert" example:"19"`                              // Steuerwert ist die aus dem Basiswert berechnete Steuer
-	Waehrung   waehrungscode.Waehrungscode `json:"waehrung,omitempty" example:"EUR" validate:"required"` // Waehrung is the currency
+	Basiswert               decimal.Decimal             `json:"basiswert" example:"100"`                              // Basiswert ist der Nettobetrag für den die Steuer berechnet wurde
+	Steuerwert              decimal.Decimal             `json:"steuerwert" example:"19"`                              // Steuerwert ist die aus dem Basiswert berechnete Steuer
+	Waehrung                waehrungscode.Waehrungscode `json:"waehrung,omitempty" example:"EUR" validate:"required"` // Waehrung is the currency
+	BasiswertVorausgezahlt  *decimal.Decimal            `json:"basiswertVorausgezahlt,omitempty"`                     // BasiswertVorausgezahlt ist die Brutto-Summe der vorausbezahlten Beträge
```

Comment:
in bo4e.net steht das sei brutto

https://github.com/Hochfrequenz/BO4E-dotnet/blob/e8b423055888b0724db392ac62b9bffab92cce98/BO4E/COM/Steuerbetrag.cs#L44

Reply:
Brutto :) https://github.com/Hochfrequenz/BO4E-dotnet/pull/258